src/peft/tuners/oft/layer.py

```diff
@@ -89,15 +297,13 @@ def __init__(self, base_layer: nn.Module, **kwargs) -> None:
         base_layer: the pretrained model layer
         """
         self.base_layer = base_layer
-        # OFT info
-        self.oft_r = nn.ParameterDict({})
-        self.oft_s = nn.ParameterDict({})
+        self.oft_R = nn.ParameterDict({})
```

Comment:
If we want to rename parameters, we would need to ensure that during model loading, names are also remapped, otherwise the change is backwards incompatible. As an example where we remapped parameter names, check this DoRA code:

https://github.com/huggingface/peft/blob/759bb70acea8486b3a493792a5b6ba80b5217fb4/src/peft/utils/save_and_load.py#L108-L119

Note that ideally, we don't rename the parameters at all, even if the new names are more descriptive.